### PR TITLE
Renewal and Watercycle, added autohide bar code, tux was not visible …

### PR DESCRIPTION
**Pull request by clock21am**
…in portrait mode

### CODE REVIEW

File: src/activities/renewable_energy/RenewableEnergy.qml [hunk 1]
```diff
@@ -41,6 +42,9 @@ ActivityBase {
         oldHeight: height
     }
 
+
```
Comment (petitlapin) [inline src/activities/renewable_energy/RenewableEnergy.qml]: don't add empty lines

File: src/activities/renewable_energy/RenewableEnergy.qml [hunk 2]
```diff
@@ -70,8 +74,11 @@ ActivityBase {
             property bool hasWon: false
         }
 
-        onStart: { }
+        onStart: { barAtStart = ApplicationSettings.isBarHidden;
```
Comment (petitlapin) [inline src/activities/renewable_energy/RenewableEnergy.qml]: go back to the line after the {

File: src/activities/renewable_energy/RenewableEnergy.qml [hunk 3]
```diff
@@ -70,8 +74,11 @@ ActivityBase {
...
+            ApplicationSettings.isBarHidden = true;
+        }
         onStop: {
+            ApplicationSettings.isBarHidden = barAtStart;
             initLevel()
```
Comment (petitlapin) [inline src/activities/renewable_energy/RenewableEnergy.qml]: it was already there but do you why init the level when leaving?

File: src/activities/renewable_energy/RenewableEnergy.qml (hunk 1 above)
Comment (clock21am) [inline src/activities/renewable_energy/RenewableEnergy.qml]: sorry, i am removing them

File: src/activities/renewable_energy/RenewableEnergy.qml (hunk 2 above)
Comment (clock21am) [inline src/activities/renewable_energy/RenewableEnergy.qml]: yes, added

File: src/activities/renewable_energy/RenewableEnergy.qml (hunk 3 above)
Comment (clock21am) [inline src/activities/renewable_energy/RenewableEnergy.qml]: i removed it, and tried to run the activity it was working fine for me

Comment (clock21am) [inline src/activities/renewable_energy/RenewableEnergy.qml]: should i remove it ?

**Comment by petitlapin** [comment]
Thanks for your patch, this is merged in KDE's git master branch

**Pull request closed without merging**